config/env.development

```diff
@@ -226,8 +226,8 @@ LOG_LEVEL=debug
 MOCK_ELASTIC=
 
 # Redis Server info, password may be optional (e.g., leave empty if you don't set one)
-#REDIS_URL=redis://127.0.0.1
-#REDIS_PORT=6379
+REDIS_URL=redis://127.0.0.1
+REDIS_PORT=6379
```

Comment:
I'm not sure if this is going to work considering how `satellite` [creates a redis client](https://github.com/Seneca-CDOT/satellite/blob/743ef81b40fb1d7f07223f78b95a7ba4690fbe0f/src/redis.js#L6).
I don't know if we should change `telescope` to adapt to `satellite` or the other way around.

cc @humphd

Reply:
I don't think we should do this, we talked about it here #2685, I must have uncommented